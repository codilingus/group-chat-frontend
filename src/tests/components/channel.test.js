import React from 'react';
import { shallow } from 'enzyme';
import Channel from '../../modules/main/navigator/channels/channel';

test('should render Channel correctly', () => {
  const wrapper = shallow(<Channel />);
  expect(wrapper).toMatchSnapshot();
});
