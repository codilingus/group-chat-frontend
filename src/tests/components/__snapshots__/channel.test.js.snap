// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Channel correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Channel />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        "# ",
        undefined,
      ],
      "className": "channel-container",
      "onClick": [Function],
    },
    "ref": null,
    "rendered": Array [
      "# ",
      undefined,
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "# ",
          undefined,
        ],
        "className": "channel-container",
        "onClick": [Function],
      },
      "ref": null,
      "rendered": Array [
        "# ",
        undefined,
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`should render Channels correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Channel />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        "# ",
        undefined,
      ],
      "className": "channel-container",
      "onClick": [Function],
    },
    "ref": null,
    "rendered": Array [
      "# ",
      undefined,
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "# ",
          undefined,
        ],
        "className": "channel-container",
        "onClick": [Function],
      },
      "ref": null,
      "rendered": Array [
        "# ",
        undefined,
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
